_posts/2016-03-18-Debezium-0-1-Released.adoc

```diff
@@ -10,6 +13,8 @@ Now the good news -- *_Debezium 0.1 is now available_* and includes several sign
 * A link:/docs/connectors/mysql/[connector for MySQL] to monitor MySQL databases. It's a Kafka Connect source connector, so simply install it into a Kafka Connect service (see below) and use the service's REST API to configure and manage connectors to each DBMS server. The connector reads the MySQL binlog and generates data change events for every committed row-level modification in the monitored databases. The MySQL connector generates events based upon the tables' structure at the time the row is changed, and it automatically handles changes to the table structures.
 * A small library so applications can link:/docs/embedded/[embed any Kafka Connect connector] and consume data change events read directly from the source system. This provides a much lighter weight system (since Zookeeper, Kafka, and Kafka Connect services are not needed), but as a consequence is not as fault tolerant or reliable since the application must maintain state normally kept inside Kafka's distributed and replicated logs. Thus the application becomes completely responsible for managing all state.
 
++++<!-- more -->+++
```

Comment:
Interesting, so is the "more" marker mandatory now? Not a problem really, so far it would show the first paragraph or so, if the marker isn't there.

Reply:
These markers were included due to the way jekyll handles post excerpts.  If the 'excerpt marker' is not included, a short excerpt will still be displayed but the formatting will be stripped. We may be able to improve, but I spent quite a bit of time on it and didn't come up with an alternate solution.

Reply:
Ok, I see. Can you log an issue for later investigation? It's not a huge problem of course.